server/components/data-table/view.njk

```diff
@@ -0,0 +1,90 @@
+{#
+  format,         // The report's data format.
```

Comment:
Minor - If this documents the params looks like we have head and row now instead of format and data.